.env

```diff
@@ -0,0 +1 @@
+VITE_BACKEND_URL="http://127.0.0.1:8000/api/v1"
```

Comment:
Don't put .env in the git. Add this file in the git ignore.

And create a new file named  .env.example where you just write the variables name of the .env file. not the values of those.

Reply:
delete this .env file from this branch. There won't be any file called .env